server/lobbyconnection.py

```diff
@@ -840,18 +845,14 @@ def on_connection_made(self, protocol: QDataStreamProtocol, peername: Address):
             "args": ["/numgames", self.player.game_count[RatingType.GLOBAL]],
             "uid": game.id,
             "mod": game.game_mode,
-            "mapname": use_map,
             # Following parameters are not used by the client yet. They are
```

Comment:
This comment block is now outdated and should probably be moved to the definition of `GameLaunchOptions`?

Reply:
I don't think so. The comment is still accurate. `name`, `init_mode` and any of the game launch options are not used by the client (and probably wont be until long after this code is deployed)